CONTRIBUTING.md

```diff
@@ -48,7 +48,7 @@ Because the first rule is to avoid breaking changes and because we want APIs to
 
 ## Coding Style
 
-REST APIs for Azure should be defined using the [OpenAPI v2](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/2.0.md) or [Cadl](https://aka.ms/cadl/rpaas-start)format.
+REST APIs for Azure should be defined using the [OpenAPI v2](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/2.0.md) or [Cadl](https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/cadl-structure-guidelines.md)format.
```

Comment:
@weshaggard is this what you want?

Reply:
I suspect this isn't the best link for coding style for now lets keep the existing link.